src/quickstart/index.md

```diff
@@ -0,0 +1,31 @@
+# Quick Start
+
+Getting started with Deis Workflow in three easy steps.
+
+1. Install CLI tools for Helm Classic and Deis Workflow
+2. Boot a Kubernetes and install Deis Workflow
+3. Deploy your first application
+
+This guide will help you setup a cluster suitable for evaluation, development and testing. When you are ready for staging and production, view our [production checklist](../managing-workflow/production-deployments.md).
+
+## Step 1: Install CLI tools
+
+For the quickstart we will [install both Helm Classic and Deis Workflow CLI](install-cli-tools.md).
+
+## Step 2: Boot Kubernetes and Install Deis Workflow
```

Comment:
Ah, I remember why now. Each provider has provider-specific docs on getting it backed by S3 or GCS. Never mind!